emulators.tex

```diff
@@ -8,14 +8,127 @@ \chapter{Emulators}
+\end{center}
+
+As you might spot quickly with this setup \specialkey{RUN STOP} \widekey{RESTORE} could
+be done single-handed.
+
+Please be aware this image shows graphics symbols being reached in the orientation the
+\megasymbolkey and \specialkey{Shift} key are positioned on a real Mega 65. On a PC
+keyboard it is crossovered, unfortunately. So, while \megasymbolkey \megakey{S} gives
+you the upper right corner of a rectangle \specialkey{Shift} \megakey{S} gives you the
+heart symbol every new C64 owner must have tried first in the eightees.
+
+For the PC styled special character keys the lower symbol is reached without the use
+of modifier keys. The upper symbol shows what you get in conjunction with the
+\specialkey{Shift} key.
+
+Xemu lets you have two joysticks (one at a time) on the numeric keypad.
+\specialkey{Enter} over there is used to toggle ports 1, 2 and to off when pressed
+repeatedly.
+
+Please be aware the emulator by default catches \megakey{F9} \megakey{F10}
+\megakey{F11} as shortcuts of its own functions. If you want those keys to be used by
+the emulated Mega 65 you need to setup an own keyboard mapping. The files are found
+in the folder for Xemu's settings as explained on page
+\pageref{sec:sdcard-settings-location}. This is done by copying
+\stw{keymap-default.cfg} into \stw{keymap.cfg} and then removing the following lines:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+XEMU-EXIT F9
+XEMU-FULLSCREEN F11
+\end{verbatim}
+\end{tcolorbox}
+
+and changing:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+F9 Unknown
+F11 Unknown
+\end{verbatim}
+\end{tcolorbox}
+
+into:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+F9 F9
+F11 F11
+\end{verbatim}
+\end{tcolorbox}
+
+Fullscreen and exiting Xemu is still available in the menu you reach by right mouse
+clicking into the emu's main window.
+
+\subsection{Apple MacOS start problems}
+
+If a message \stw{macOS cannot verify that this app is free from malware} pops up you
+may want control-click to start it: \index{Apple} \index{macOS}
+
+\begin{itemize}
+  \item Open Finder on your Mac.
+  \item Point your mouse to Xemu's app icon.
+  \item Control-click it to open the shortcut menu.
+  \item Now click \stw{Open} to run Xemu. By doing it this way no warning will be shown.
+\end{itemize}
+
+\subsection{SD Card, settings location}
+\label{sec:sdcard-settings-location}
+\index{SD card}
+
+You may need to go to Xemu's settings folder in order to remap your keyboard if you want
+to and to backup your SD Card emulation image file.
+
+\begin{center}
+  \includegraphics[width=0.5\linewidth]{images/xemusettings.png}
+\end{center}
+
+If you already uninstalled it or want to do it by hand.
+
+\begin{itemize}
+    \item On Windows goto File Explorer and copy the following into the address
```

Comment:
Just say "Windows," "Linux," and "macOS" followed by a colon then the path.

Reply:
@dansanderson Are you aware not every Windows user knows how to use a directory path containing embedded environment variables? I wanted to explicitly pinpoint them to Explorer's address bar.

```tex
    \item On Windows goto File Explorer and copy the following into the address
    bar: \stw{\%appdata\%$\textbackslash$xemu-lgb$\textbackslash$mega65}
```

I also wanted to follow the explanatory style of the whole manual. But I now deleted it.

Reply:
This is a good point and maybe your way is better. I've done this in other (non-MEGA65) manuals just by listing paths, but if you think more instruction is needed then feel free to use your version here.

Reply:
I made a slight adjustment.